docs/self-hosted-appcircle/self-hosted-runner/runner-vm-setup.md

```diff
@@ -512,6 +512,68 @@ tart clone macOS_240514 vm01
 
 In docker terminology, `vm01` and `vm02` will be our docker images. We will configure them separately, persist our changes and then create containers to execute build pipelines. On every build, fresh containers will be used for both runners.
 
+### Configure Runner VM Resources
+
+You can adjust the resources limits for Runner VMs based on your needs.
+
+:::info 
+By default, our runner images are configured with an 8GB memory limit and 4 CPU cores.
+:::
+
+:::caution 
+Total allocated resources should not exceed host machine's physical capacity for optimal performance.
+
+For example:
+
+| Host Memory Size | Runner Memory Config. |
+|------------------|-----------------------|
+| 8GB              | 1 x VM with 8GB       |
+| 16GB             | 1 x VM with 16GB      |
```

Comment:
Also can you switch the order of "Total 16GB x 1 16GB" and "Total 16GB x 2 8GB"? Because most of the users will go with "Total 16GB x 2 8GB".